### PR TITLE
[RB] Updated migration file

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes

`2019-07-04-remove_header_row_from_parameter_file_and_convert_back_to_Value_field_to_text.sql` had not been applied.

### CODE REVIEW

File: raisinbread/migration.md [hunk 1]
```diff
@@ -41,11 +41,18 @@
...
+2019-07-04-remove_header_row_from_parameter_file_and_convert_back_to_Value_field_to_text.sql
 2019-07-10-subproject-session-FK.sql
 2019-07-17_remove_mri_acquisition_dates_table.sql
-2019_07_04_remove_header_row_from_parameter_file_and_convert_back_to_Value_field_to_text.sql
+2019-08-05-add_projects_to_users.sql
+2019-10-01_Rename-media-column.sql
+2019-10-05-Add_alias_to_projects.sql
+2019-11-01-Add_data_release_permissions.sql
+2019-11-12-Rename_modules_QC_and_DTH.sql
+
```
Comment (PapillonMcGill) [inline raisinbread/migration.md]: @ridz1208 Does it matter if all theses lines appear in one continuous line? (same for previous release).

Comment (ridz1208) [reply raisinbread/migration.md]: not really no, the migration file is just something I use to keep track with what has been run.

the format of it does not affect anything

File: raisinbread/RB_files/RB_parameter_type.sql [hunk 2]
```diff
@@ -237,7 +237,6 @@ INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`,
...
 INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`, `RangeMin`, `RangeMax`, `SourceField`, `SourceFrom`, `SourceCondition`, `Queryable`, `IsFile`) VALUES (236,'magnetic_field_strength','text','magnetic_field_strength magically created by NeuroDB::File',NULL,NULL,NULL,'parameter_file',NULL,0,0);
 INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`, `RangeMin`, `RangeMax`, `SourceField`, `SourceFrom`, `SourceCondition`, `Queryable`, `IsFile`) VALUES (237,'dicom_0x0051:el_0x1016','text','dicom_0x0051:el_0x1016 magically created by NeuroDB::File',NULL,NULL,NULL,'parameter_file',NULL,0,0);
-INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`, `RangeMin`, `RangeMax`, `SourceField`, `SourceFrom`, `SourceCondition`, `Queryable`, `IsFile`) VALUES (238,'header','text','header magically created by NeuroDB::File',NULL,NULL,NULL,'parameter_file',NULL,0,0);
```
Comment (driusan) [inline raisinbread/RB_files/RB_parameter_type.sql]: Is this line supposed to be removed? The description just says it's upgrading the migration file.

Comment (ridz1208) [reply raisinbread/RB_files/RB_parameter_type.sql]: yes, it was a missed patch that I run

**Pull request merged**